README.rst

```diff
@@ -68,10 +73,10 @@ Contents of requirements.txt
     wheel==0.23.0
     Yarg==0.1.9
     docopt==0.6.2
-    
+
 Why not pip freeze?
 -------------------
 
-- ``pip freeze`` only saves the packages that are installed with ``pip install`` in your environment. 
-- ``pip freeze`` saves all packages in the environment including those that you don't use in your current project (if you don't have ``virtualenv``).
+- ``pip freeze`` only saves the packages that are installed with ``pip install`` in your environment.
+- ``pip freeze`` saves all packages in the environment including those that you don't use in your current project. (if you don't have ``virtualenv``).
```

Comment:
Sorry for being annoying, I swear this is the last change I will request haha but could you remove the dot on this line? I think it is an artifact from the master branch